verticapy/tests_new/machine_learning/vertica/__init__.py

```diff
@@ -17,48 +17,497 @@
+        ),
+        "aic": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "bic": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "F-statistic": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "Prob (F-statistic)": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "Kurtosis": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "Skewness": {"rel": 1e-03, "abs": ABS_TOLERANCE},
+        "Jarque-Bera (JB)": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "df": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "ss": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "ms": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "f": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "p_value": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "coef_": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "intercept_": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "score": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "predict": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "to_python": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "load_model": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+    },
+    "RandomForestClassifier": {
+        "auc": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "prc_auc": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "accuracy": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "log_loss": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "precision": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "recall": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "f1_score": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "mcc": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "markedness": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "csi": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "confusion_matrix": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "cutoff_curve": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "lift_chart": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "prc_curve": {"rel": 1e-02, "abs": ABS_TOLERANCE},
+        "predict_proba": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "roc_curve": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "score": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "predict": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "to_python": {"rel": REL_TOLERANCE, "abs": ABS_TOLERANCE},
+        "load_model": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+    },
+    "DecisionTreeRegressor": {
+        "explained_variance": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "max_error": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "median_absolute_error": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "mean_absolute_error": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        "mean_squared_error": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "mean_squared_log_error": {"rel": 1e-00, "abs": ABS_TOLERANCE},
+        "rmse": {"rel": 1e-01, "abs": ABS_TOLERANCE},
+        **dict.fromkeys(["r2", "R-squared"], {"rel": 1e-00, "abs": ABS_TOLERANCE}),
```

Comment:
R-squared tolerance of 1? That should not be the case. It is too huge.

Reply:
same as first comment.